CONTRIBUTING.md

```diff
@@ -68,14 +68,24 @@ First, ensure dependencies are installed
 - [gpgme](https://www.gnupg.org/related_software/gpgme/) (on macOS, `brew install gpgme`)
 - [dlv](https://github.com/go-delve/delve) (for debugging) - Because this project uses go modules, install this globally with `go get github.com/go-delve/delve` to run the tool outside of the project directory.
 
+## CLI debugging
+
 Build a binary with
 
 `go build -o opctl-beta ./cli`
```

Comment:
the cli/.opspec/compile op completes pretty fast these days; I wonder if it's worth pointing to use that since it might make for fewer moving parts.

Reply:
That's fine, just a note that this is specifically under the section "# Local development with native go tooling"